extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+        },
+        "countries": {
+          "bufferView": 4,
+          "arrayOffsetBufferView": 5,
+          "stringOffsetBufferView": 6
+        }
+      }
+    }
+  }
+}
+```
+
+### Group Metadata
+
+Metadata may be assigned to groups. Groups represent collections of contents. Contents are assigned to groups with the `3DTILES_metadata` content extension.
+
+Group metadata can be paired with [`3DTILES_multiple_contents`](../../3DTILES_multiple_contents/0.0.0) to assign multiple contents in a tile to different groups.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "layer": {
+            "properties": {
+              "color": {
+                "type": "ARRAY",
+                "componentType": "UINT8",
+                "componentCount": 3
+              },
+              "priority": {
+                "type": "UINT32"
+              }
+            }
+          }
+        }
+      },
+      "groups": {
+        "buildings": {
+          "class": "layer",
+          "properties": {
+            "color": [128, 128, 128],
+            "priority": 0
+          }
+        },
+        "trees": {
+          "class": "layer",
+          "properties": {
+            "color": [10, 240, 30],
+            "priority": 1
+          }
+        }
+      }
+    }
+  },
+  "root": {
+    "refine": "ADD",
+    "geometricError": 32768.0,
+    "boundingVolume": {
+      "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+    },
+    "children": [
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "buildings.b3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "buildings"
+            }
+          }
+        }
+      },
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "trees.i3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "trees"
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+### Feature Metadata
+
+A feature is an entity that has both geometry and metadata, such as a 3D building in a city. Features are the finest level of granularity within the tileset.
+
+Metadata may be assigned to features using the glTF extension [`EXT_feature_metadata`](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0).
+
+Feature metadata classes may be included in the `3DTILES_metadata` schema. This lets an application know what classes exist before requesting content.
+
+### Statistics
+
+Statistics provide aggregate information about entities in a tileset. Statistics are provided on a per-class basis.
+
+* `count` is the number of entities that conform to the class
```

Comment:
I'm still not sure I understand the vocab decision for Entity as an instance of a Class, but is "conform" the right word for instance here?  I'm not a Programming Languages researcher, but I'm pretty sure this would read something like "count is the number of **objects** that are **instances** of the **class**" or something like that using generic programming language type system terms